src/dpp/discordclient.cpp

```diff
@@ -622,6 +622,15 @@ discord_client& discord_client::connect_voice(snowflake guild_id, snowflake chan
 			return *this;
 		}
 	}
+
+	if (enable_dave) {
+		/* Disable dave if we're connecting to stage channel */
+		auto *c = find_channel(channel_id);
```

Comment:
I don't think this is a good idea, there's nothing special about stage channels that needs Dave disabled it's just the Dave pr isn't finished.

regardless this assumes channel cache is enabled which isn't going to work if you disable cache.